bullet_train/docs/webhooks/outgoing.md

```diff
@@ -27,6 +27,8 @@ payment.generate_webhook(:succeeded)
 ## Delivery
 Webhooks are delivered asyncronously in a background job by default. If the resulting HTTP request results in a status code other than those in the 2XX series, it will be considered a failed attempt and delivery will be reattempted a number of times.
 
+All of the delivery attempts and events are stored in the database. There is a background job to purge records in these tables as they can grow large in size over time. You can schedule this job to purge records older than 90 days by default.
+
```

Comment:
Can we add an example of at least how to call the job here, just so people don't have to go hunting for it?